File: app/src/main/java/com/pritam/githubexplorer/GithubExplorerApp.java
package com.pritam.githubexplorer;

import android.content.Context;

import androidx.multidex.MultiDex;
import androidx.multidex.MultiDexApplication;

public class GithubExplorerApp extends MultiDexApplication {

    @Override
    protected void attachBaseContext(Context base) {
        super.attachBaseContext(base);
        MultiDex.install(this);
    }
}

File: app/src/main/java/com/pritam/githubexplorer/GithubExplorerApp.kt
package com.pritam.githubexplorer

import android.app.Application
import android.content.Context
import androidx.multidex.MultiDex

class GithubExplorerApp : Application() {

    override fun onCreate() {
        super.onCreate()
        MultiDex.install(this)
    }

    override fun attachBaseContext(base: Context?) {
        super.attachBaseContext(base)
        MultiDex.install(this)
    }
}